index.js

```diff
@@ -320,7 +320,7 @@ let Fixtures = {
             let request = {
                 url: '/' + VERSION + '/' + model.module,
                 method: 'POST',
-                data: model.attributes || {},
+                data: _.cloneDeep(model.attributes) || {},
```

Comment:
Thinking a little more about it, maybe I'll split the [new test](https://github.com/sugarcrm/thorn/pull/134/files#diff-4663341696a08eef2606d83e7fb282a9R183) into two: 
1. it should only fill required fields when respective values aren't supplied
2. it should not update given fixtures if required fields aren't supplied

I'll do it tomorrow morning and I'll solve the conflicts with the latest changes that got in today.